.github/workflows/ci.yaml

```diff
@@ -182,10 +182,10 @@ jobs:
     env:
       CI: false # react-scripts build treats warnings as errors if this is true
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
         with:
-          node-version: 20.11.1
+          node-version-file: '.nvmrc'
           cache: yarn
       - run: yarn set version 3.2.1
```

Comment:
Does the `actions/setup-node` installation of Node come with Corepack?

If so, we might be able to replace this with `corepack enable yarn` and have it read the Yarn version from `package.json` 🤔

(Disclaimer: I’m not saying this from experience; it just seems like something that _should_ work 😅)